assets/javascripts/running.js

```diff
@@ -171,8 +175,9 @@ function updateTestStatus(newStatus) {
         reloadBrokenThumbnails(true);
       }
     })
-    .fail(function () {
+    .catch(error => {
       console.log('ERROR: modlist fail');
```

Comment:
Do you need this as there is a console.error in the next line?

Reply:
It is good to see where it failed and afterwards also what the error was.